openquake/hmtk/seismicity/declusterer/dec_afteran.py

```diff
@@ -159,6 +159,14 @@ def decluster(self, catalogue, config):
 
         return vcl, flagvector
 
+    @staticmethod
```

Comment:
If this method is used only for the 2D declustering, you can remove the depths since they are not used, or (perhaps even better) you can directly use the haversine function.